docs/source/technical_tutorials/search/maap_stac_r.ipynb

```diff
@@ -0,0 +1,734 @@
+{
```

Comment:
<p>Same thing for this cell, I would remove the control flow that catches collections with no items. That might be helpful for the focused search section, but not here since these collections should all have items!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/464/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>